packages/api/src/controllers/farmExpenseTypeController.js

```diff
@@ -156,6 +159,8 @@ const farmExpenseTypeController = {
         }
 
         data.expense_translation_key = baseController.formatTranslationKey(data.expense_name);
+        //prevent empty strings
+        data.custom_description = data.custom_description || undefined;
```

Comment:
**This is actual buggy behaviour, it will need fixing**

The user needs to be able to delete an existing description by clearing that text area, but using `undefined` means that the original description will always persist once it's set. It needs to be set explicitly to `null`.

If done in the frontend:
```jsx
  const onSubmit = (payload) => {
    dispatch(
      updateCustomExpenseType({
        ...payload,
        custom_description: payload.custom_description || null,
        expense_type_id,
      }),
    );
  };
  ```

Or leaving logic where it is:
```suggestion
        data.custom_description = data.custom_description || null;
```